src/debug.ts

```diff
@@ -0,0 +1,35 @@
+import { app, dialog } from 'electron'
+import { is } from 'electron-util'
+import appConfig from 'electron-settings'
+import electronDebug from 'electron-debug'
+
+export const CONFIG_KEY = 'debug-mode'
+const OPTIONS = {
+  showDevTools: false
+}
+
+export function showRestartDialog(enabled: boolean): void {
+  const state = enabled ? 'enable' : 'disable'
+
+  dialog.showMessageBox(
+    {
+      type: 'info',
+      buttons: ['Restart', 'Cancel'],
+      message: 'Restart required',
+      detail: `To ${state} debug mode, please restart Gmail Desktop`
+    },
+    response => {
```

Comment:
Type missing?

Reply:
dialog.showMessageBox provides the type of number without us needing to explicitly state it.